parsing/pprintast.mli

```diff
@@ -24,3 +24,5 @@ val pattern: Format.formatter -> Parsetree.pattern -> unit
 val signature: Format.formatter -> Parsetree.signature -> unit
 val structure: Format.formatter -> Parsetree.structure -> unit
 val string_of_structure: Parsetree.structure -> string
+
+val longident : Format.formatter -> Longident.t -> unit
```

Comment:
I think that `longident` could be at the top rather than the bottom of the interface since it prints simpler items.

Reply:
Will do -- the order in the other declarations is not the one you suggest, so I will reorder everything.